.eslintrc.json

```diff
@@ -1,25 +1,27 @@
 {
-    "env": {
-      "browser": true,
-      "es6": true,
-      "jest": true
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
     },
```

Comment:
- [ ] Kindly ensure that the config rules here match exactly what is set out [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json). It's important you use the same rules shared by other students too.

Reply:
Thank you @mikenath223